### DISCUSSION TITLE
can't see so-* indices from kibana

### DISCUSSION THREAD

**Discussion by ziko0079** [Security-Onion-Solutions/securityonion#2685] [Unsupported Versions] [answered]
Hi all
When i try to see the current indices from kibana or using cli i don't get the so-* indices
bellow is the output I get : 

[elasticsearch@elasticsearch ~]$ curl -GET 'localhost:9200/_cat/indices'
green open elastalert_status_status  cMtIHB6HTzam59TLy4FRJw 1 0    0 0   208b   208b
green open elastalert_status         MBzsUKY-RC6ErSjkDRiIXQ 1 0    0 0   208b   208b
green open elastalert_status_past    H2xIU2FxTvWZ6YS3oOlPpg 1 0    0 0   208b   208b
green open elastalert_status_silence aUbx9uHdS9CbpM_mPDBNDw 1 0    0 0   208b   208b
green open elastalert_status_error   N45MAps7R6qrHSuysG4j6w 1 0    0 0   208b   208b
green open .kibana_1                 X-PkJRUyRby-CEj_MitGfg 1 0 2314 4  1.1mb  1.1mb
green open .tasks                    xmMFxM1WRJqlXwcqqm64pQ 1 0    9 0 34.8kb 34.8kb

so I m wondering where the data indices are located?

**Comment by TOoSmOotH** [comment, marked as answer]
https://github.com/Security-Onion-Solutions/securityonion/discussions/1720

**Reply by ziko0079** [reply]
Sorry for the lack of information.
I am using  a distributed deployment using ISO  Version 2.3.21 
so-status on the manager is showing that everything is ok except suricata, in the logs  I see that it 's because it can't find bound0 interface but we are getting the alerts in the the web interface from the forward node, on the forward node and search node everything is OK,
in the Kibana interface we see logs from suricata ,zeek, strelka .... but when trying to see the actual indices status we don't see so-* indices.
Bellow the output from Curl in the manager : 
curl -GET 'localhost:9200/_cat/indices'
green open elastalert_status_status cMtIHB6HTzam59TLy4FRJw 1 0 0 0 208b 208b
green open elastalert_status MBzsUKY-RC6ErSjkDRiIXQ 1 0 0 0 208b 208b
green open elastalert_status_past H2xIU2FxTvWZ6YS3oOlPpg 1 0 0 0 208b 208b
green open elastalert_status_silence aUbx9uHdS9CbpM_mPDBNDw 1 0 0 0 208b 208b
green open elastalert_status_error N45MAps7R6qrHSuysG4j6w 1 0 0 0 208b 208b
green open .kibana_1 X-PkJRUyRby-CEj_MitGfg 1 0 2314 4 1.1mb 1.1mb
green open .tasks xmMFxM1WRJqlXwcqqm64pQ 1 0 9 0 34.8kb 34.8kb

we are getting same results from kibana web interface.
Thank you

**Reply by weslambert** [reply]
The indices will be held on the storage nodes, so you will need to look there, as opposed to the manager if you want to see it from that perspective.  The default install uses cross cluster search to search from the manager across all of the independent clusters/nodes.

**Reply by ziko0079** [reply]
Thank you very much for your reply
